x-pack/plugins/cases/server/routes/api/comments/get_all_comment.ts

```diff
@@ -27,11 +28,12 @@ export const getAllCommentsRoute = createCasesRoute({
     try {
       const caseContext = await context.cases;
       const client = await caseContext.getCasesClient();
+      const res: attachmentDomainV1.Attachments = await client.attachments.getAll({
```

Comment:
Is this right to be a domain type? Just thinking out loud here about the logic. Should APIs only deal with `MyTypeRequest` and `MyTypeResponse`?

Reply:
A lot of our APIs return a response that either is a domain type or an array of domain types. Maybe we could create a type alias like `type GetAllAttachmentResponse = Attachments`.